server/products.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const db = require('APP/db');
+const router = require('express').Router();
+const Product = db.model('products');
+
+router.get('/', function(req, res, next) {
+  // res.send('Hellooooooo');
+  Product.findAll()
```

Comment:
for cleanliness, I'd delete the comment on line 8 -- but as you go on to have more complicated routes, maybe add comments about what the routes are hitting / used for.